packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DestinationPicker.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { PaginateControl } from '../../views/Controls/Paginate';
+import {
+  EmptyMessageControl,
+  LoadingControl,
+  MessageControl,
+} from '../Controls';
+import { ViewElement } from '../../context/elements';
+import { displayText } from '../../displayText/en';
+import { useDestinationPicker } from './CopyView/useDestinationPicker';
+import { CLASS_BASE } from '../constants';
+import { DataTableControl } from '../../controls/DataTableControl';
+import { ControlsContextProvider } from '../../controls/context';
+import {
+  getDestinationListFullPrefix,
+  getDestinationPickerTableData,
+} from './utils/getDestinationPickerDataTable';
+import { ControlsContext } from '../../controls/types';
+import { Breadcrumb } from '../../components/BreadcrumbNavigation';
+import { DescriptionList } from '../../components/DescriptionList';
+const {
+  actionSetDestination,
+  actionDestinationPickerCurrentFolderSelected,
+  actionDestinationPickerNoMoreFolders,
+  actionDestinationPickerDefaultError,
+} = displayText;
+
+const DEFAULT_PAGE_SIZE = 10;
+export const DEFAULT_LIST_OPTIONS = {
+  pageSize: DEFAULT_PAGE_SIZE,
+  delimiter: '/',
+};
+
+export const DestinationPicker = ({
```

Comment:
Is this a akin to a view? As it stands, it seems to operate as one but the naming very much implies it is just a stray component